lib/pulsar/client/transport/curl.py

```diff
@@ -1,11 +1,5 @@
 import logging
 
-try:
-    from galaxy import eggs
-    eggs.require("six")
```

Comment:
Actually this is a leftover from your commit https://github.com/galaxyproject/galaxy/commit/033b1e88da99b86ffbd33321374aa59d1d72484c#diff-8ce0f1f93264f339e0bf364c7c19d3bd :laughing: